apps/ecosystem/src/ecosystem.repository.ts

```diff
@@ -87,6 +87,63 @@ export class EcosystemRepository {
     }
   }
 
+   //eslint-disable-next-line camelcase
+    async addOrganizationInEcosystem(orgs: IEcosystemOrgs[], orgIds: string[], ecosystemId: string): Promise<[Prisma.BatchPayload, IEcosystemOrgsData[]]> {
+    try {
+      const result = await this.prisma.$transaction([
+
+      this.prisma.ecosystem_orgs.createMany({
+        data: orgs
+      }),
+
+      this.prisma.ecosystem_orgs.findMany({
```

Comment:
@bhavanakarwade Create separate method for this query for re-usability.

Reply:
Resolved